tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "typeRoots": [ "<types-directory-created-in-#1>", "./node_modules/@types"],
```

Comment:
by default the types in `node_modules/@types` are added, updating `typeRoots` might add other types as well. Can I please know if `<types-directory-created-in-#1>` in `typeRoots`?

Reply:
Not sure how it was added maybe as a part of some node modules updates. Have tried running the app without it and it works well hence, removed the typeRoots Compiler option.